Region/Region.cc

```diff
@@ -348,12 +349,26 @@ casacore::WCRegion* Region::MakeEllipseRegion(const std::vector<CARTA::Point>& p
         casacore::Vector<casacore::Quantum<casacore::Double>> center(2);
         center(0) = casacore::Quantity(world_coords(0), coord_units(0));
         center(1) = casacore::Quantity(world_coords(1), coord_units(1));
+	
         // make Quantities for ellipsoid radii
         casacore::Vector<casacore::Quantum<casacore::Double>> radii(2);
         radii(0) = casacore::Quantity(bmaj, "pix");
         radii(1) = casacore::Quantity(bmin, "pix");
 
-        ellipse = new casacore::WCEllipsoid(center, radii, _xy_axes, _coord_sys);
+	// Convert theta to a Quantity
+	casacore::Quantity quantityTheta = casacore::Quantity(static_cast<double>(theta), "rad");
```

Comment:
In new code style, variable names are lower_case: quantity_theta, major_axis, minor_axis